### PR TITLE
[qt-advanced-docking-system] updated qt-advanced-docking-system to 3.2.1

### PR DESCRIPTION
**Pull request by stijnherfst**
Updates the library to the newest release which fixes several bugs and adds features.

### CODE REVIEW

**Comment by stijnherfst** [comment]
I'm not sure why only the Linux version is saying that there is no CMakeLists.txt present while the Windows one can find it fine and I made no big changes to the project.

File: ports/qt-advanced-docking-system/portfile.cmake [hunk 1]
```diff
@@ -3,8 +3,8 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
```
Comment (NancyLi1013) [inline ports/qt-advanced-docking-system/portfile.cmake]: `include(vcpkg_common_functions)` is no longer needed.
Could you please remove it?

File: ports/qt-advanced-docking-system/portfile.cmake [hunk 2]
```diff
@@ -18,8 +18,8 @@ vcpkg_configure_cmake(
 
 vcpkg_install_cmake()
 
-file(INSTALL ${SOURCE_PATH}/LICENSE.md DESTINATION ${CURRENT_PACKAGES_DIR}/share/qt-advanced-docking-system RENAME copyright)
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/qt-advanced-docking-system RENAME copyright)
```
Comment (NancyLi1013) [inline ports/qt-advanced-docking-system/portfile.cmake]: Could you please update 
`file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/qt-advanced-docking-system RENAME copyright)`
as
`file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)`?

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by NancyLi1013** [comment]
Hi @stijnherfst 

Can you install this port on Linux platform successfully on your local?

File: ports/qt-advanced-docking-system/CONTROL [hunk 3]
```diff
@@ -1,4 +1,4 @@
 Source: qt-advanced-docking-system
-Version: 2019-08-14
+Version: 3.2.1
 Build-Depends: qt5-base, zlib, bzip2
```
Comment (NancyLi1013) [inline ports/qt-advanced-docking-system/CONTROL]: Could you please add `homepage `here?
Sorry, I missed this last time.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by stijnherfst** [comment]
It seems like this might be an issue on your end with qt5-base indeed not compiling properly.

**Comment by NancyLi1013** [comment]
We have been working on the regreassions on Linux about `libhdfs3 `,`osg-qt` and try to fix them now.
But I'm not sure if this PR is realted to these regressions.
Once fixed, I will rerun this PR.
Let's wait for the new CI test results.

**Comment by NancyLi1013** [comment]
The regressions on Linux like this:
```
CMake Error at /ci/myagent/_work/1/s/installed/x64-linux/share/cmake/Qt5Gui/Qt5GuiConfig.cmake:15 (message):
  The imported target "Qt5::Gui" references the file

     "/ci/myagent/_work/3/s/installed/x64-linux/lib/libEGL.so"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "/ci/myagent/_work/1/s/installed/x64-linux/share/cmake/Qt5Gui/Qt5GuiConfigExtras.cmake"

  but not all the files it references.
```
Since this error is related to `qt5-base`.
Due to conflicts with `angle` and `qt5-base`, `ms-angel:x64-linux` is set to fail now.
So currently, we can add `qt-advanced-docking-system:x64-linux=fail` to ci.baseline.txt.

**Comment by BogdanW3** [comment]
Just to note, building under Linux works for me now so I think the update is ok.
Homepage line is now included too.

**Comment by stijnherfst** [comment]
This issue should automatically be resolved at some point in the future and thus I think merging it should be fine.

**Comment by stijnherfst** [comment]
So is this going to be merged? The problem with compilation on Linux seems to be due to another package that this depends on having issues and is not related to this package then. The previous version of this package should then also fail to build because one of its dependencies fails and thus this PR doesn't regress anything.

**Review by dan-shaw — APPROVED** [review]

**Comment by dan-shaw** [comment]
The changes look good to me. Right now, none of the Qt packages are getting downloaded on our CI for some reason. We're looking into it and trying to get it resolved ASAP. After that, we can merge this. I will tag this 'reviewed' so we don't forget about it.

**Comment by dan-shaw** [comment]
@stijnherfst Can you resolve the conflict? we just fixed the Qt problem in our CI

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
@stijnherfst 
Thanks for your update.
LGTM now. Once the regression about `qt5-tools` on Linux is fixed, I will tag this as 'Reviewed'.

**Comment by stijnherfst** [comment]
That's an issue on your end, right?

**Comment by ras0219-msft** [comment]
LGTM, thanks for the PR!

**Pull request merged**